.github/workflows/diff-change-to-dist.yaml

```diff
@@ -42,18 +42,15 @@ jobs:
           github-token: ${{secrets.GITHUB_TOKEN}}
           script: |
             const fs = require('fs').promises
+            const commentOnPr = require('./shared/stats/comment.js').commentOnPr
             const diff = await fs.readFile(
               process.env.GITHUB_WORKSPACE + '/dist.diff', 'utf8'
             )
 
-            const commentText = '## Changes to dist\n' +
+            const commentText = '<!-- Diff release -->' +
+              '## Changes to dist\n' +
```

Comment:
**issue** I think this needs a line break after the comment to make sure Github interprets the next line as Markdown. It displays as `## Changes to dist` in the [latest comment on this PR](https://github.com/alphagov/govuk-frontend/pull/3571#issuecomment-1539670369).

**suggestion** It's also be neat to move the comment part inside `commentOnPr` so that comment + line break is hidden from whatever script is trying to comment and that script just needs to provide a simple string ("Diff release" in this case) as a marker 😊

Reply:
Thanks for the suggestions!

I've shifted the comment into the function itself, so just pass in a simple string.

I've also popped a SHA on the comment to help us identify what the comment applies to.

I had a glance at testing - it's easy to mock the `context` object, but the Octokit/Github API is a bit trickier. There's a couple of ready-made mock Github APIs out there (https://github.com/kiegroup/mock-github), or we could try the fixtures route (https://github.com/octokit/fixtures) - don't see a premade scenario for comment actions, though.